### ISSUE TITLE
[Anon] Bug report: Escher.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/necromunda-underhive#3]
**File:** Escher.catz

**BattleScribe version:** 2.01.14

**Platform:** iPhone / iPod / iPad

**Dropbox:** No

**Description:** 2 bugs:
can’t find a way to select abilities for leader/champion
can’t equip with special weapons the single ganger allowed to do so when uou create a gang

**Comment by pinecones** [comment]
I will check it out and update.

**Comment by pinecones** [comment]
Abilities like skills are available under character. Thinking of renaming and regrouping.

**Comment by pinecones** [comment]
Upon review of rules I cannot see any mention of a ganger allowed to equip status weapons. I think this is being confused with gang war. Underhive is much more restrictive and simple in options (hence my first project).

**Comment by asteroid1984** [comment]
you’re right, my bad, i was creating a gang with gang war system and didn’notice i could click on tenunit name to add abilities. are you goong to create a gang war repository as well?

**Comment by pinecones** [comment]
No probs. I am working on a gang war at the moment. It is on my personal page currently, and I only have Escher and Goliath working. It does have features like campaign tracking functions. 

And you can select some special weapons. It is hard to limit selections other than a hard limit, which doesn't work when you have to have X gangers for y other guys. So some honesty and knowing the structure is required. But no more than pen and paper I suppose ;)

I'll be adding it to bsdata when I get home today. So if you can wait until then it will be migrated shortly.

**Comment by asteroid1984** [comment]
that’s great news and thanks for the great job!
